assests/javascript/index.js

```diff
@@ -5,23 +5,30 @@ const addTaskIcon = document.getElementById("add-task-icon");
 const addTaskBox = document.getElementById("add-task-box");
 const addTaskInputErrorMassage = document.getElementById("title-error");
 const networkErrorMassage = document.getElementById("network-error");
+const taskList = document.getElementById("task-list");
+const filterTasksView = document.getElementById("filter-tasks");
 
 addTaskInput.addEventListener(
   "keydown",
   (event) => {
     if (event.key === "Enter") {
       handleNewTask(addTaskInput.value);
+      addTaskInput.value = "";
```

Comment:
Now it's time to make a function for it.

Reply:
done